src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,71 @@
+package duke.storage;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.Scanner;
+import duke.task.*;
```

Comment:
I think the imported classes should be listed explicitly as per the coding standard given?
```suggestion
import duke.task.Deadline;
import duke.task.Event;
import duke.task.Task;
import duke.task.TaskList;
import duke.task.ToDo;
```